keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -234,19 +236,17 @@ No, upgrading to a version of the Kubelet with this feature will not enable the
 
 ###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
 
-Yes. Roll back and remove the `--config-dir` flag from the kubelet's CLI, as well as the KubeletDropInConfig featuregate.
-
+Yes. To disable the feature, roll back by removing the --config-dir flag from the kubelet's CLI.
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 
-This feature will be re-enabled via adding back the `--config-dir` flag to the CLI, as well as the KubeletDropInConfig featuregate, as mentioned above.
-
+This feature will be re-enabled via adding back the `--config-dir` flag to the CLI.
 
 ###### Are there any tests for feature enablement/disablement?
 
 A test will be added to assemble a single, functional kubelet configuration object from various individual drop-in config files.
 
-A test will be added to check that if a user attempts to use the `--config-dir` flag without the featuregate, the kubelet will properly error.
+A test will be added to check that if a user attempts to use the `--config-dir` flag without the `KUBELET_CONFIG_DROPIN_DIR_ALPHA` environment variable set, the kubelet will properly handle the configuration and provide appropriate feedback.
```

Comment:
this is not applicable in beta, as we're going to drop the env field. you can remove this paragraph

Reply:
Fixed